addons/mail/models/discuss/discuss_channel.py

```diff
@@ -242,9 +249,10 @@ def create(self, vals_list):
             # is_pinned + ensure they have rights to see channel
             if not self.env.context.get('install_mode') and not self.env.user._is_public():
                 partner_ids_to_add = list(set(partner_ids + [self.env.user.partner_id.id]))
-            vals['channel_member_ids'] = membership_ids_cmd + [
-                (0, 0, {'partner_id': pid})
-                for pid in partner_ids_to_add if pid not in membership_pids
+            vals["channel_member_ids"] = membership_ids_cmd + [
+                (0, 0, {"partner_id": pid, "is_channel_admin": pid == self.env.user.partner_id.id})
```

Comment:
Shouldn't this be replaced by `Command.create` now that we have some diff on it?